content/docs/manuals/worker-configuration.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Worker Configuration"
+summary: ""
+draft: false
+weight: 302000000
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+The Oakestra worker is managed by two components:
+
+* The **NodeEngine** is responsible for worker ↔ cluster communication
+* The **NetManager** is responsible for worker ↔ worker communication
+
+The NodeEngine regularly collects system information and sends it to the cluster manager. It is also responsible for handling deployment commands.
+Only the NodeEngine is required to register a worker with a cluster, however the NetManager is required for service-to-service communication through the overlay network.
+
+### Useful References
+
+{{< card-grid >}}
+{{< link-card
+  title="Set up a Worker"
+  description="Connect your first worker to an Oakestra cluster"
+  href="../../getting-started/oak-environment/add-edge-devices-workers-to-your-setup/"
+  target="_blank"
+>}}
+{{< link-card
+  title="Worker Architecture"
+  description="Explore the components of an Oakestra worker"
+  href="../../concepts/high-level-architecture/#worker-node"
+  target="_blank"
+>}}
+{{< link-card
+  title="Overlay Network"
+  description="Learn more on the overlay network"
+  href="../networking-internals/overlay-network"
+  target="_blank"
+>}}
+{{< /card-grid >}}
+
+## Installation
+
+You can install the latest bundled NodeEngine and NetManager with
+
+```bash
+curl -sfL oakestra.io/install-worker.sh | sh - 
+```
+
+{{< callout context="note" title="Specify the Version" icon="outline/info-circle" >}}
+You can specify which version should be installed from the [list](https://github.com/oakestra/oakestra-net/releases) by first setting
+
+```bash
+export OAKESTRA_VERSION=<release version>
+```
+
+and then re-running the above install command.
+{{< /callout >}}
+
+## Configuration
+
+The NodeEngine and NetManager are implemented as systemd services. While it is possible to run the former without the latter, this configuration is impractical for most applications, therefore the NodeEngine service internally calls on the NetManager service. The NetManager cannot be run without the NodeEngine.
```

Comment:
```suggestion
The NodeEngine and NetManager are implemented as systemd services. The NodeEngine service internally calls the NetManager service. The NetManager cannot be run without the NodeEngine.
```

Personal recommendation: Remove the 'what not to do' sentence and focus on the 'how to do it right' (IMO: otherwise this confuses the reader more that it helps to avoid mistakes)